### PR TITLE
Introduced full mutation testing

### PR DESCRIPTION
**Pull request by Ocramius**
3e7fcd

### CODE REVIEW

**Comment by Ocramius** [comment]
"You need to install and enable xdebug in order to allow for code coverage generation."

Dang.

File: src/ProxyManager/Exception/FileNotWritableException.php [hunk 1]
```diff
@@ -39,24 +39,4 @@ public static function fromInvalidMoveOperation(string $fromPath, string $toPath
             $toPath
         ));
     }
-
-    public static function fromNonWritableLocation($path) : self
```
Comment (Ocramius) [inline src/ProxyManager/Exception/FileNotWritableException.php]: Actually, as a better alternative, I should deprecate this method, then restore the tests and drop it in 3.0.

**Comment by samsonasik** [comment]
@Ocramius I created PR to your PR at #349 to install xdebug manually at php 7.1 env

**Comment by Ocramius** [comment]
Waiting for @travis-ci, then merging and possibly releasing.

**Comment by Ocramius** [comment]
\o/

**Pull request merged**